src/js/edu-benefits/1990n/components/IntroductionPage.jsx

```diff
@@ -34,7 +33,7 @@ class IntroductionPage extends React.Component {
                   <span>You won’t be able to save your work or come back to finish. So before you start, it’s a good idea to gather information about your education history and the school you want to attend.</span>
                 </div>
               </div>
-              <p>We strongly encourage you to work with a trained professional who can help you pick the right program. <a href="/disability-benefits/apply/help/index.html">Find an accredited representative.</a> Or call 888-442-4551 (888-GI-BILL-1) from 8:00 a.m. to 7:00 p.m. (ET) to talk with an education specialist.</p>
```

Comment:
In the original issue, we had the two lines `Remove "Find a representative section"` and `Remove VSO sentence`. Does removing this line take care of both of those?